### PR TITLE
hotfix: parent issue

### LINKED ISSUE #268: Disable Bounties for "Parent Issues"
### Context

It's fast and easy to write todo lists in the original comment of a GitHub issue, and then immediately convert them to "child issues" directly from the GitHub user interface.

The problem I am seeing is that the bounty system breaks down where contributors can "double dip" on rewards easily by claiming bounties for the actual bounties (child issues) and then the "meta" or "parent" issue.

The argument possibly could be made for project management. However in the future all of the **issue creators**, and **commenters** (both are how the project manager would "manage") are to receive compensation for their contributions upon the completion of the bounty.

### Proposal

- When changing a label, the bot should check if there are linked issues in the original comment and disable setting a price (and most importantly, as a consequence, payment permit generation.)

- The bot should comment explaining why it refuses to set a price.

---

              Parent issues should not have bounties. Instead, the child issues should have bounties.

_Originally posted by @pavlovcik in https://github.com/ubiquity/grants/issues/5#issuecomment-1519067539_

### PR DESCRIPTION
**Pull request by wannacfuture**
Resolves #268

### CODE REVIEW

File: src/handlers/pricing/action.ts [hunk 1]
```diff
@@ -13,6 +13,16 @@ export const pricingLabelLogic = async (): Promise<void> => {
   if (!payload.issue) return;
   const labels = payload.issue.labels;
 
+  logger.info(`Checking if the issue is parent`);
```
Comment (pavlovcik) [inline src/handlers/pricing/action.ts:16]:
```suggestion
  logger.info(`Checking if the issue is a parent issue.`);
```

**Review by pavlovcik — CHANGES_REQUESTED** [review]
Would you mind linking the original context/conversation for this hotfix in the opening comment on the pull request?

**Comment by wannacfuture** [comment]
REady for QA, @web4er 
@pavlovcik

**Comment by wannacfuture** [comment]
Seems the upsertComment function is not appropriate for this condition.

**Comment by pavlovcik** [comment]
> Would you mind linking the original context/conversation for this hotfix in the opening comment on the pull request?

It's not clear to me why there are three hot fix pull requests from you for this issue.

**Comment by pavlovcik** [comment]
Would somebody be so kind as to QA this?

**Review by web4er — APPROVED** [review]
> Would somebody be so kind as to QA this?

QA: https://github.com/web4erOrg/ubiquibot/issues/26

Working well.

**Review by pavlovcik — APPROVED** [review]
Good QA

**Comment by wannacfuture** [comment]
> > Would you mind linking the original context/conversation for this hotfix in the opening comment on the pull request?
> 
> It's not clear to me why there are three hot fix pull requests from you for this issue.

haha it would be only one hot fixes if the previous PRs hasn't been merged accidentialy by someone, @pavlovcik 
lol 😂

**Pull request merged**